src/AppBundle/Security/RateLimiter.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Security;
+
+use AppBundle\Entity\IpRequest;
+use Doctrine\ORM\EntityManager;
+
+class RateLimiter
+{
+    private $em;
+
+    private $ipRequest;
+
+    const AGGREGATION_DELAY = 5;
+    const MAX_ATTEMPTS = 15;
```

Comment:
I'm wondering if these two previous parameters could not be added to the `config.yml` file in a static parameters section.

We could override them in the `config_test.yml` file so we can set super-high values and get rid of the injection of the current environment.